src/go/k8s/tests/e2e-unstable/decommission/README.txt

```diff
@@ -0,0 +1,8 @@
+This test
+
+0. enables downscaling
```

Comment:
with 22.3.0 - should we allow to always enable downscaling? It should be safe now, right?

Reply:
Seems reasonable to me, but let's do this as a stand-alone issue. I'd also like to add the option of scaling up the statefulset during pod rolls to keep the replication levels at the expected levels. That would require a little rework of the operator but it would also require some way of informing an autoscaler. It might make sense to do the downscale change as part of that work.

We also don't have any check to see that we even can downscale safely. What if you have 3 redpanda nodes and replica 3 topics? Shouldn't we block downscaling to 2?

Reply:
yeah we discussed this several times already. I think the best option would be for these safety checks to be inside redpanda but I think there was some pushback...